local/debian/Makefile

```diff
@@ -0,0 +1,24 @@
+IMG ?= saucelabs/forwarder:debian-systemd
+CONTAINER ?= forwarder-debian
```

Comment:
How about

IMG = debian-systemd:bullseye
CONTAINER = saucelabs/forwarder-testing-debian

Reply:
CONTAINER could be `forwarder-testing-debian` as it must match ` [a-zA-Z0-9][a-zA-Z0-9_.-]*:`